company-tng.el

```diff
@@ -132,6 +140,11 @@ confirm the selection and finish the completion."
   (setq company-frontends '(company-tng-frontend
                             company-pseudo-tooltip-frontend
                             company-echo-metadata-frontend))
+  (setq company-clang-insert-arguments nil
+        company-semantic-insert-arguments nil
+        company-rtags-insert-arguments nil
+        lsp-enable-snippet nil)
```

Comment:
Maybe you should also disable company-lsp-enable-snippet? This way even if LSP sends a snippet, company-lsp won't insert it.

Reply:
According to Ivan, this variable is no longer used. But if you notice a problem that is fixed with setting it to nil, please let me know.